telemetry_core/src/test/java/com/newrelic/telemetry/metrics/MetricBufferTest.java

```diff
@@ -38,6 +38,19 @@ void testMetrics() {
     assertEquals(expectedGauge, metricBuffer.getMetrics().poll());
   }
 
+  @Test
+  @DisplayName("Get size of Metrics")
+  void testMetricsSize() {
+    Attributes expectedCommonAttributes =
+        new Attributes().put("key1", "value1").put("key2", true).put("key3", 123.456);
+
+    MetricBuffer metricBuffer = new MetricBuffer(expectedCommonAttributes);
```

Comment:
can you add an assertion that the size is `0` before you add one? Then, this looks great. Thanks!